clusterloader2/pkg/framework/framework.go

```diff
@@ -155,6 +163,26 @@ func (f *Framework) DeleteAutomanagedNamespaces() *errors.ErrorList {
 	return errList
 }
 
+// DeleteStaleAutomanagedNamespaces deletes all stale automanged namespaces.
+func (f *Framework) DeleteStaleAutomanagedNamespaces(namespaces []string) *errors.ErrorList {
```

Comment:
Rename to `DeleteNamespace` - this function is not limited to automated and stale namespaces and could be used in other places.